Packs/CommonScripts/Scripts/ArrayToCSV/README.md

```diff
@@ -13,7 +13,7 @@ Converts a simple Array into a textual comma separated string
 
 | **Argument Name** | **Description** |
 | --- | --- |
-| value |  |
+| value |  An array of stringsfor example: `["abc","def"]` |
```

Comment:
Please re-generate the file using the yml. Also notice this parameter can also be a long string and not just an array, when running using the CLI, for example:
"val1, val2, val3" will generate the same result.